pkg/operator/ceph/cluster/mon/drain.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2021 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mon
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/rook/rook/pkg/clusterd"
+	policyv1beta1 "k8s.io/api/policy/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	monPDBName = "rook-ceph-mon-pdb"
```

Comment:
Nit: I would call this `rook-ceph-mons`. The type is already a PDB, so it becomes redundant information to put `pdb` in the name of a PDB resource.

@travisn @leseb do you have other thoughts?

Reply:
If this is not displayed to the user then I'm fine with `rook-ceph-mons`.

Reply:
Agreed that name would be better, but I would suggest we wait for that in a separate PR. Let's keep this PR simpler to get it in sooner. If we rename it, then we do have to worry about deleting a legacy pdb.

Reply:
Yes, if we change then we need to delete the old mon PDBs as well. Keeping it `rook-ceph-mon-pdb` for now.